style/js/shield_defs.js

```diff
@@ -1431,7 +1414,7 @@ export function loadShields(shieldImages) {
     },
   };
 
-  shields["US:TX"] = roundedRectShield("white", "black", "black", 1, 1, null);
+  shields["US:TX"] = shields["default"];
   shields["US:TX:Andrews:Andrews:Loop"] = banneredShield(
     roundedRectShield("white", "#003f87", "#003f87", 1, 1, null),
```

Comment:
Let's change this one to a 2px radius too. Same for the rest of the rectangles defined for Texas, Wyoming and Nova Scotia.

Reply:
Fixed in 6f5db25